classes/TranslatableBehavior.php

```diff
@@ -297,7 +297,7 @@ public function lang($context = null)
      * Checks if this model has transatable attributes.
      * @return true
      */
-    public function hasTransatableAttributes()
+    public function hasTranslatableAttributes()
```

Comment:
You need to update the call to the method here as well: https://github.com/rainlab/translate-plugin/blob/e603c487fa0c1c4a9ca31cd6c1a9305ea2364389/classes/EventRegistry.php#L84 do a search of the code to see if it was misspelled anywhere else too.

Reply:
Never mind, I see you fixed it in https://github.com/rainlab/translate-plugin/pull/363/files#diff-252810a1b0bce82b1a1ef724cee3c8deR96